documentation/api/query/v4/upgrading-from-v3.markdown

```diff
@@ -83,6 +83,10 @@ Each change below is marked with the corresponding release version.
 
 * (3.0) `/v4/reports/<hash>/events` This convenience endpoint allows you to show events for a particular report by its hash. See the [/v4/reports documentation](./reports.html)
 
+* (3.0) `/v4/reports/<hash>/metrics` This convenience endpoint allows you to show metrics for a particular report by its hash. See the [/v4/reports documentation](./reports.html)
```

Comment:
this isn't a convenience endpoint for hsqldb users, it's their only option. Might be worth mentioning that in these notes.